src/css/modules/_panel.scss

```diff
@@ -211,6 +211,55 @@
   }
 }
 
+.panel__simplepage {
```

Comment:
It's hard for me to see what's being done here because I can't see the design, but it does seem like a lot of overriding. I'd probably split some of this logic out into separate components. For instance, if you need a 45em centered content area, then that could be a `.panel__small` component that you apply to the p tag. In general, this many tag rules under one class seems a little strange.